packages/components/src/components/video/video.stories.ts

```diff
@@ -0,0 +1,194 @@
+};
+
+export default {
+  title: 'Components/sd-video',
+  component: 'sd-video',
+  args,
+  argTypes,
+  parameters: { ...parameters },
+  decorators: [withActions] as any
+};
+
+/**
+ * The `sd-video` component can be used to wrap external video elements in order to provide basic styling for Union Investment.
+ */
+
+export const Default = {
+  render: (args: any) => {
+    return generateTemplate({
+      constants: [
+        {
+          type: 'slot',
+          name: 'default',
+          value:
+            '<div style="width: 400px; height: 225px;" class="flex items-center justify-center overflow-hidden"><img src="./placeholders/generic.jpg" /></div>'
+        }
+      ],
+      args
+    });
+  }
+};
+
+/**
+ * `sd-video` with all possible property configurations.
+ */
+
+export const PlayingOverlay = {
+  name: 'Playing × Overlay',
+  render: (args: any) => {
+    return html`
+      ${generateTemplate({
+        axis: {
+          x: { type: 'attribute', name: 'playing' },
+          y: { type: 'attribute', name: 'overlay' }
+        },
+        constants: [
+          {
+            type: 'slot',
+            name: 'default',
+            value:
+              '<div style="width: 400px; height: 225px;" class="flex items-center justify-center overflow-hidden"><img src="./placeholders/generic.jpg" /></div>'
+          }
+        ],
+        args
+      })}
+    `;
+  }
+};
+
+// Script used in next 2 stories
+const videoElementScript = html`<script>
+  function runScriptOnPathChange() {
+    // *NOTE: Storybook single page behavior complicates queries between the "Docs" and "Video Element" stories.
+    // Therefore it is necessary to query for all existing elements and target the one specific to our story, otherwise we end up with strange overlapping behaviors in Storybook.
+    const videoEls = document.querySelectorAll('#videoExample');
+    const sdVideos = document.querySelectorAll('#sdVideoExample > sd-video');
+
+    const videoEl = document.querySelectorAll('#videoExample')[1] || document.querySelectorAll('#videoExample')[0];
+    const sdVideo =
+      document.querySelectorAll('#sdVideoExample > sd-video')[1] ||
+      document.querySelectorAll('#sdVideoExample > sd-video')[0];
+
+    function setupEventListeners() {
+      sdVideo?.addEventListener('sd-play', playVideo);
+      videoEl?.addEventListener('play', updatePlayingStatus);
+      videoEl?.addEventListener('pause', updatePlayingStatus);
+    }
+
+    function playVideo() {
+      videoEl.play();
+    }
+
+    function updatePlayingStatus() {
+      sdVideo.playing = videoEl.paused ? false : true;
+    }
+
+    function removeEventListeners() {
+      // ensure we affect ANY existing elements by querying for all, see *NOTE above
+      const videoEls = document.querySelectorAll('#videoExample');
+      const sdVideos = document.querySelectorAll('#sdVideoExample > sd-video');
+      const sdVideoPlayButtons = document.querySelectorAll('#sdVideoExample > sd-video');
```

Comment:
Could you add some more documentation for the users of our library?

Pointing them to what's important for them etc. Some stuff isn't important, like Storybook changes, maybe you can clarify that, too.

So more thinking from a user of our library: What is importatnt. What do I have to do why. What's not important.

Reply:
You mean via code comments in this block?

Reply:
Yes